PProPanel/ext.js

```diff
@@ -222,11 +222,11 @@ function loadJSX() {
 	var extensionPath = csInterface.getSystemPath(SystemPath.EXTENSION);
 
 	// load general JSX script independent of appName
-	const extensionRootGeneral = extensionPath + '/jsx/';
+	var extensionRootGeneral = extensionPath + '/jsx/';
```

Comment:
> Have you checked out the TypeScript sample yet?

No, I didn't check it out yet. Personally, I don't like to work with TypeScript, but I will have a look at the sample in this repo.

For my panel projects, I've set up a boilerplate using [create-react-app](https://github.com/facebookincubator/create-react-app), because reactJS is the ui framework we use at work.

> Chrome 27 - const is recognized only when not in strict mode. No block scoping.
> Chrome 41 - const has block scoping in strict mode, but no block scoping otherwise.

Hmm... the fact that ES specs differ between versions of CEP really just drives me to hold on to the extra compile step using babel, as this will maximize compatibility and it allows me to use the new cool things of JavaScript like arrow functions, spread operator and so on ;-)